packages/react-reconciler/src/ReactFiberCommitHostEffects.js

```diff
@@ -214,6 +219,38 @@ function getHostParentFiber(fiber: Fiber): Fiber {
   );
 }
 
+export function commitFragmentInstanceInsertionEffects(fiber: Fiber): void {
+  let parent = fiber.return;
+  while (parent !== null) {
+    if (isFragmentInstanceParent(parent)) {
+      const fragmentInstance: FragmentInstance = parent.stateNode;
+      commitNewChildToFragmentInstance(fiber.stateNode, fragmentInstance);
```

Comment:
You expect this to be an Element but this Fiber can be either a HostText or HostComponent so it can also be a Text. Because you call `commitFragmentInstanceInsertionEffects` for both HostText and HostComponent.

Currently you don't have any methods that operate on Text Nodes so you could probably skip HostText or call another Config that is a noop for now. Like maybe React Native could focus HostText?